action.yml

```diff
@@ -16,7 +16,7 @@ inputs:
     required: false
 runs:
   using: "docker"
-  image: docker://ghcr.io/actionshub/publish-gem-to-github:v1.0.0
+  image: docker://ghcr.io/actionshub/publish-gem-to-github:pr-10
```

Comment:
This looks wrong mate

Reply:
This is only for the PR. It gets fixed on main again.